project/MiMaFilters.scala

```diff
@@ -2,19 +2,26 @@
 import com.typesafe.tools.mima.core._
 
 object MiMaFilters {
-  val Library: Seq[ProblemFilter] = Seq(
-    // New API in 3.4.X
-    ProblemFilters.exclude[ReversedMissingMethodProblem]("scala.quoted.Quotes#reflectModule#FlagsModule.AbsOverride"),
-    ProblemFilters.exclude[ReversedMissingMethodProblem]("scala.quoted.Quotes#reflectModule.ValOrDefDefTypeTest"),
-    ProblemFilters.exclude[ReversedMissingMethodProblem]("scala.quoted.Quotes#reflectModule.ValOrDefDefMethods"),
-    ProblemFilters.exclude[ReversedMissingMethodProblem]("scala.quoted.Quotes#reflectModule#defnModule.FunctionClass"),
-    ProblemFilters.exclude[ReversedMissingMethodProblem]("scala.quoted.Quotes#reflectModule#defnModule.PolyFunctionClass"),
-    ProblemFilters.exclude[ReversedMissingMethodProblem]("scala.quoted.Quotes#reflectModule#TypeReprMethods.dealiasKeepOpaques"),
-    ProblemFilters.exclude[ReversedMissingMethodProblem]("scala.quoted.Quotes#reflectModule#SymbolMethods.paramVariance"),
-    ProblemFilters.exclude[ReversedMissingMethodProblem]("scala.quoted.Quotes#reflectModule#TypeLambdaMethods.paramVariances"),
+  val LibraryBackwards: Map[String, Seq[ProblemFilter]] = Map(
+    // In general we should never have backwards incompatible changes in the library.
+    // Only exceptional cases should be added here.
+    Build.previousDottyVersion -> Seq(
+      // This language feature was in 3.4.0-RC1 but will be removed in 3.4.0-RC2
+      ProblemFilters.exclude[MissingFieldProblem]("scala.runtime.stdLibPatches.language#deprecated.ascriptionVarargsUnpacking"),
+      ProblemFilters.exclude[MissingClassProblem]("scala.runtime.stdLibPatches.language$deprecated$ascriptionVarargsUnpacking$"),
+    )
+  )
+  val LibraryForward: Map[String, Seq[ProblemFilter]] = Map(
+    // Additions that require a new minor version of the library
+    Build.previousDottyVersion -> Seq(
+      // This language feature is not in 3.4.0-RC1 but will be added in 3.4.0-RC2
+      ProblemFilters.exclude[MissingFieldProblem]("scala.runtime.stdLibPatches.language.3.5-migration"),
+      ProblemFilters.exclude[MissingFieldProblem]("scala.runtime.stdLibPatches.language.3.5"),
+      ProblemFilters.exclude[MissingClassProblem]("scala.runtime.stdLibPatches.language$3$u002E5$"),
+      ProblemFilters.exclude[MissingClassProblem]("scala.runtime.stdLibPatches.language$3$u002E5$minusmigration$"),
+    )
```

Comment:
With `3.4.0-RC2`, the two lists `LibraryBackwards`/ `LibraryForward` should be empty.